webapp/src/app.tsx

```diff
@@ -2111,13 +2111,23 @@ export class ProjectView
     completeTutorial() {
         pxt.tickEvent("tutorial.complete");
         core.showLoading("leavingtutorial", lf("leaving tutorial..."));
-        this.exitTutorialAsync()
-            .then(() => {
-                let curr = pkg.mainEditorPkg().header;
-                return this.loadHeaderAsync(curr);
-            }).done(() => {
-                core.hideLoading("leavingtutorial");
-            })
+
+        if (pxt.BrowserUtils.isIE()) {
```

Comment:
we should consider have a single behavior for all browser. e.g. exit tutorial takes you to the home screen

Reply:
Add it to the UI meeting discussion list :smile:. I also prefer going straight to the home screen